src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -332,6 +333,11 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('expiration_in_days', options_list=['--expiration-in-days', c.deprecate(target='--days', redirect='--expiration-in-days', hide=True)],
                    help='Number of days for which the credentials will be valid. If not specified, the expiration will default to the max value "9999-12-31T23:59:59.999999+00:00"', type=int, required=False)
 
+    for scope in ['acr token create', 'acr token credential generate']:
+        with self.argument_context(scope) as c:
+            c.argument('expiration', validator=validate_expiration_time,
```

Comment:
seems there're different expiration unit/measure: expire in days, expire at exact date etc.

Reply:
consider `expire_at` as option name?

Reply:
@yungezz, Yes `expiration` is for exact date, the `expiration-in-days` is for days to make the input easier.

@fengzhou-msft, I name it as `expiration` for wording consistency with `--expiration-in-days`; or I can do `--expire-in-days` and `--expire-at`, let me know.

Reply:
I think `--expire-in-days` and `--expire-at` is more clear.

Reply:
@yugangw-msft let me know whether you would make the change. Either is fine as users can always know the meaning of the options from help messages.